rust-toolchain.toml

```diff
@@ -1,3 +1,3 @@
 [toolchain]
-channel = "1.76.0"
+channel = "1.78.0"
```

Comment:
`cargo make` only supports `>1.78.0`. Although there are more recent Rust versions, I wanted to make the minimum version bump required in this PR to play it safe